app/controllers/registrations_controller.rb

```diff
@@ -57,7 +57,7 @@ def after_inactive_sign_up_path_for(resource)
   end
 
   def check_registration_enabled
-    render_404 and return if Fulcrum::Application.config.fulcrum.disable_registration
```

Comment:
Doesn't this return necessary?

Reply:
I don't think so, but let me change the code

Reply:
```ruby
def check_registration_enabled
  return unless Fulcrum::Application.config.fulcrum.disable_registration

  render_404
end
```